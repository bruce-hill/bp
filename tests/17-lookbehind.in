1234
xxxx23yy
3
xxx3xx
