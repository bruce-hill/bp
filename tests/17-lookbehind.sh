# <pat is a lookbehind
# Example: bp -p '<`: word' will match words that come after a colon
bp -p '<`2 `3'
