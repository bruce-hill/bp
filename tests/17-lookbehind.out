1234
xxxx23yy
